eng/pipelines/common/evaluate-default-paths.yml

```diff
@@ -68,6 +67,8 @@ jobs:
     - subset: installer
       include:
       - docs/manpages/*
+      - src/native/common/*
```

Comment:
This is not needed either. You can see this comment to see how the tooling fully works: https://github.com/dotnet/runtime/blob/main/eng/pipelines/evaluate-changed-paths.sh#L2-L17